config/src/test/kotlin/org/springframework/security/config/annotation/web/FormLoginDslTests.kt

```diff
@@ -90,6 +89,32 @@ class FormLoginDslTests {
                 }
     }
 
+    @Configuration
+    @EnableWebMvc
+    @EnableWebSecurity
+    open class DisabledConfig {
+        @Bean
+        open fun securityFilterChain(http: HttpSecurity): SecurityFilterChain {
+            http.formLogin()
+            http {
+                formLogin {
+                    disable()
+                }
+            }
+            return http.build()
+        }
+    }
+
+    @Test
+    fun `request when disabled does not provide login page`() {
```

Comment:
To clarify, could this test be named instead `request when formLogin disabled does not provide login page`?

Reply:
Done